src/focus.service.ts

```diff
@@ -521,11 +521,10 @@ export class FocusService {
       return false;
     }
 
-    if (el.tabIndex > -1) {
-      return true;
+    if (el.tabIndex < 0) {
+      return false;
     }
```

Comment:
Okay, that is really bizarre.  Unfortunately I can't really think of a clean solution that lets us keep within the standards while allowing a Edge/Xbox to do its thing. So yea, I guess the solution in this PR of just adding back manual checking is fine. Let's also check for `getAttribute('tabIndex') >= 0`  though to pick up on manually-defined tabIndicies though.